tests/CecilTests/CompilerTestBase.cs

```diff
@@ -15,19 +14,27 @@ public partial class CompilerTestBase
         static CecilTypeSystem CreateCecilTypeSystem()
         {
             var self = typeof(CompilerTestBase).Assembly.GetModules()[0].FullyQualifiedName;
-            #if USE_NETSTANDARD_BUILD
+#if USE_NETSTANDARD_BUILD
             var selfDir = Path.GetDirectoryName(self);
             var selfName = Path.GetFileName(self);
             self = Path.GetFullPath(Path.Combine(selfDir, "../netstandard2.0/", selfName));
-            #endif
+#endif
             var refsPath = self + ".refs";
             var refs = File.ReadAllLines(refsPath).Concat(new[] {self});
             return new CecilTypeSystem(refs, null);
         }
 
         public CompilerTestBase() : this(CreateCecilTypeSystem())
         {
-            
+#if NETFRAMEWORK
+            // TODO: It's the hack for VS tests
+            var selfDirectory = Path.GetDirectoryName(typeof(CompilerTestBase).Assembly.Location);
+            AppDomain.CurrentDomain.AssemblyResolve += (sender, args) =>
+            {
+                var name = args.Name.Split(',').First() + ".dll";
```

Comment:
You do realize that .First() can be replaced with indexer access, right?

Reply:
Yes, but it seems to me that it will be more readable, and besides, it will [still happen](https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Linq/src/System/Linq/First.cs#L59)

Reply:
Fix to indexer access